scripts/scil_compute_ssst_frf.py

```diff
@@ -92,6 +94,13 @@ def main():
     assert_inputs_exist(parser, [args.input, args.bvals, args.bvecs])
     assert_outputs_exist(parser, args, args.frf_file)
 
+    if len(args.roi_radii) == 1:
+        roi_radii = args.roi_radii[0]
+    elif len(args.roi_radii) == 3:
+        roi_radii = args.roi_radii
+    else:
+        parser.error('--roi_radii cannot be of size (2,).')
```

Comment:
Maybe say 'wrong size for --roi_radii.'? As the size could be anything else than 1 or 3.

Reply:
Fixed, let me know what you think

Reply:
Looks good to me!